lib/client-api/src/story_store.js

```diff
@@ -89,7 +89,15 @@ export default class StoryStore extends EventEmitter {
     this._selection = data === undefined ? this._selection : { storyId, viewMode };
     this._error = error === undefined ? this._error : error;
 
-    setTimeout(() => this.emit(Events.STORY_RENDER), 1);
+    setTimeout(() => {
+      // preferred method to emit event.
+      if (this._channel) {
+        this._channel.emit(Events.STORY_RENDER);
+       }
+     
+      // should be deprecated in future.
+      this.emit(Events.STORY_RENDER);
```

Comment:
does this need to be in an `else` block?

Reply:
@benoitdion, I have no clue to be fair, thats why I've asked @ndelangen for his opinion.

Reply:
NO, no else here,

`this.emit()` is currently required, the other does nothing.

But in the future, the reserve will be true.

Reply:
> But in the future, the reserve will be true.

cryptic 👀